src/app/common/resolvers/dataset-view/dataset-settings/dataset-settings-transform-tab.resolver.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright Kamu Data, Inc. and contributors. All rights reserved.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the LICENSE file.
+ */
+
+import { inject } from "@angular/core";
+import { ResolveFn } from "@angular/router";
+import { combineLatest, map } from "rxjs";
+import { AppConfigService } from "src/app/app-config.service";
+import { DatasetService } from "src/app/dataset-view/dataset.service";
+import { DatasetSubscriptionsService } from "src/app/dataset-view/dataset.subscriptions.service";
+import { NavigationService } from "src/app/services/navigation.service";
+import { datasetSettingsActiveSectionResolver } from "../dataset-settings-active-section.resolver";
+import { isSettingsTabAccessibleHelper } from "src/app/dataset-view/additional-components/dataset-settings-component/dataset-settings.helpers";
+import { SettingsTabsEnum } from "src/app/dataset-view/additional-components/dataset-settings-component/dataset-settings.model";
+import { DatasetBasicsFragment, DatasetMetadata } from "src/app/api/kamu.graphql.interface";
+
+export const datasetSettingsTransformTabResolver: ResolveFn<DatasetBasicsFragment | null> = (route, state) => {
+    const datasetService = inject(DatasetService);
+    const datasetSubService = inject(DatasetSubscriptionsService);
+    const navigationService = inject(NavigationService);
+    const appConfigService = inject(AppConfigService);
+    const activeTab = datasetSettingsActiveSectionResolver(route, state) as SettingsTabsEnum;
+
```

Comment:
Again, you are perfectly aware about your current tab in this resolver.

Reply:
Removed